tutorials/developing-services-with-k8s.md

```diff
@@ -37,9 +36,7 @@ In this tutorial, we're going to use the [Guestbook](https://cloud.google.com/co
 
 To set up your laptop, you'll need to install a few basic components.
 
-First, you'll want to install Docker. If you don't have Docker installed, follow the [install instructions](https://www.docker.com/community-edition).
-
-Next, we're going to want to install the `gcloud` and `kubectl` commands. Follow the instructions at [https://cloud.google.com/sdk/downloads](https://cloud.google.com/sdk/downloads]) to download and install the Cloud SDK. Then, insure `kubectl` is installed:
+First, we're going to want to install the `gcloud` and `kubectl` command line tool. Follow the instructions at [https://cloud.google.com/sdk/downloads](https://cloud.google.com/sdk/downloads]) to download and install the Cloud SDK. Then, insure `kubectl` is installed:
```

Comment:
I suggest dropping the phrase "we're going to want to," and "tool" should be plural, so:

First, install the \`gcloud\` and \`kubectl\` command line tool**s**.

Also, looks like there's an extra `]` after *.../sdk/downloads* so the link is broken.